package.json

```diff
@@ -17,6 +17,8 @@
     "lint": "yarn lint:js && yarn lint:style"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.30",
```

Comment:
I don't recommend pulling all of fontawesome so we can get the fa-play icon

Reply:
Alright